lib/model/rwfolder.go

```diff
@@ -87,6 +89,8 @@ type rwFolder struct {
 	copiers      int
 	pullers      int
 	shortID      uint64
+	newVersion   protocol.Vector
```

Comment:
Ok but I'm still not sure how to "pass in" cur.version and file.version.  It's not calling the method/function in the usual way.  And it's calling several other functions in that way as well.  I don't know if changing that will compromise the structure of other code that could lead to worse outcomes.

`		if ok && p.inConflict(cur.Version, file.Version) {
		// There is a conflict here. Move the file to a conflict copy instead
		// of deleting. Also merge with the version vector we had, to indicate
		// we have resolved the conflict.
		file.Version = file.Version.Merge(cur.Version)
		err = osutil.InWritableDir(p.moveForConflict, realName)`